guide/command-handling/adding-features.md

```diff
@@ -283,6 +283,9 @@ module.exports = {
 	},
 };
 ```
+::: tip
+You Can Export Anything In Module.exports and call it in help
```

Comment:
If anything, this would have to not be overcapitalized like that.
Additionally, exported object is available not only in help command, so i think this sounds slightly more fitting.
```suggestion
You can export anything in `module.exports` object and access it later.
```
On the other hand, "anything" may not exactly be true either, but that would depend on reader.

Reply:
Sounds Good!